src/app/rpmostree-builtin-upgrade.cxx

```diff
@@ -58,9 +60,44 @@ static GOptionEntry option_entries[] = {
   { "unchanged-exit-77", 0, 0, G_OPTION_ARG_NONE, &opt_unchanged_exit_77, "If no new deployment made, exit 77", NULL },
   { "trigger-automatic-update-policy", 0, G_OPTION_FLAG_HIDDEN, G_OPTION_ARG_NONE, &opt_automatic, "For automated use only; triggered by automatic timer", NULL },
   { "lock-finalization", 0, G_OPTION_FLAG_HIDDEN, G_OPTION_ARG_NONE, &opt_lock_finalization, "Prevent automatic deployment finalization on shutdown", NULL },
+  { "force", 'f', 0, G_OPTION_ARG_NONE, &opt_force, "Force an upgrade even if an updates driver is registered.", NULL},
   { NULL }
 };
 
+/* Get the `Documentation` property `update_driver_sd_unit`. Documentation is returned 
+ * as a string array GVariant in `unit_doc_array`. */
+static gboolean
+get_update_driver_doc (RPMOSTreeSysroot *sysroot_proxy,
+                       const char       *update_driver_sd_unit,
+                       GVariant        **unit_doc_array,
+                       GCancellable     *cancellable,
+                       GError          **error)
+{
+  GDBusConnection *connection =
+    g_dbus_proxy_get_connection (G_DBUS_PROXY (sysroot_proxy));
```

Comment:
Very minor nit but I'd pass the `GDBusConnection *connection` as a parameter to the function (and invoke `g_dbus_proxy_get_connection()` at the call site); conceptually it doesn't have anything to do with the sysroot.